README.Rmd

```diff
@@ -181,15 +180,17 @@ head(pop)
 
 ```{r}
 # create samples
-samples <- draw_samples(pop, 3, c(1, 10, 50, 100))
+samples <- draw_samples(pop, 100, c(1, 10, 50, 100))
 head(samples)
 ```
 ```{r}
 # plot sample histogram
 plot_sample_hist(pop, samples, height, c(10, 50, 100))
 ```
 
-
+```{r}
+plot_sampling_hist(samples, height, c(10, 50), 100)
+```
```

Comment:
I think it would be nice to show them as a grid since the output isn't very intuitive...do you think it would be possible to add a couple of lines here putting them into a grid for the README display and maybe we can try to revise the function for the next milestone?
(Note: I did this by making and testing the function that generates a list of histograms and then have the user-facing function be the one that plots them in a grid calling the other function. We can maybe have a chat about this next week :) )